.github/workflows/build.yml

```diff
@@ -73,29 +90,33 @@ jobs:
         apt install -y build-essential pkg-config make
 
     - name: Install macOS Dependencies
-      if: contains(matrix.os, 'macos') && 0
-      run: |
-        set -x
-
-    - name: Setup Compilers
+      if: contains(matrix.os, 'macos') && matrix.compiler == 'flang'
       run: |
         set -x
-        if test "$FC" = "flang" ; then \
-          echo "FC=flang-new" >> "$GITHUB_ENV" ; \
-        else \
-          echo "FC=gfortran-${GCC_VERSION}" >> "$GITHUB_ENV" ; \
-        fi
-        if test "$CC" = "gcc" ; then echo "CC=gcc-${GCC_VERSION}" >> "$GITHUB_ENV" ; fi
-        if test "$CXX" = "g++" ; then echo "CXX=g++-${GCC_VERSION}" >> "$GITHUB_ENV" ; fi
+        brew update
+        brew install llvm@${GCC_VERSION} flang
```

Comment:
Why is `GCC_VERSION` used in `llvm@${GCC_VERSION}`?  Should there be a defined `LLVM_VERSION that is used here instead?

Reply:
That was just me being lazy and re-using the variable, but I can switch things up to clarify

Reply:
Fascinating. I'm surprised it works. Yes, I'd recommend switching it.

Reply:
Resolved in 5a3cba49